airbyte-integrations/connectors/source-postgres/src/test-integration/resources/postgresql.conf

```diff
@@ -0,0 +1,783 @@
+# -----------------------------
```

Comment:
I copied this file here cause intellij would not run the `CdcPostgresSourceStandardTest` without this